src/main/resources/react/src/routes/routes.js

```diff
@@ -1,16 +1,25 @@
-import React from "react";
-import { Route, Switch } from "react-router-dom";
-import HomePage from "../pages/homepage/HomePage.js";
-import Auth from "../pages/FormRegister/Auth.js";
-import UserInfo from "../pages/UserInfo/UserInfo";
+import React from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+import HomePage from '../pages/homepage/HomePage.js';
+import Auth from '../pages/FormRegister/Auth.js';
+import UserInfo from '../pages/UserInfo/UserInfo';
 
 export default () => {
-  return (
-    <Switch>
-      <Route path="/" component={HomePage} exact />
-      <Route path="/userInfo/" component={UserInfo} />
-      <Route path="/registration/" component={Auth} />
-      <Route path="/registration/register" exact component={Auth} />
-    </Switch>
-  );
+	const { isAuthenticated } = useSelector( state => state.auth );
+
+	return isAuthenticated ? (
+		<Switch>
+			<Route path="/" component={ HomePage } exact/>
+			<Route path="/userInfo/" component={ UserInfo }/>
+			<Redirect to={ '/' }/>
+		</Switch>
+	) : (
+		<Switch>
+			<Route path="/" component={ HomePage } exact/>
+			<Route path="/logIn/" component={ Auth }/>
```

Comment:
You have misspelling here into "logIn". All letters must be low case.

Reply:
Done